src/parse/state/post.js

```diff
@@ -27,7 +27,7 @@ function boundary(spec, species) {
   }
 
   const headerValue = [...contentType].find((t) =>
-    t.value.includes('multipart/form-data')
+    (t.value || []).includes('multipart/form-data')
```

Comment:
It would be preferred to default to the same data type as the check is ran on for clarity (even though includes is both on String and Array prototypes). `t.value || ""`